v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dbutils/connection/CassandraConnectionHelper.java

```diff
@@ -136,7 +136,8 @@ public boolean isConnectionPoolInitialized() {
    */
   private CqlSession createCqlSession(CassandraShard cassandraShard) {
     CqlSessionBuilder builder = CqlSession.builder();
-    DriverConfigLoader configLoader = cassandraShard.getConfigLoader();
+    DriverConfigLoader configLoader =
+        CassandraDriverConfigLoader.fromOptionsMap(cassandraShard.getOptionsMap());
     configLoader.getInitialConfig();
```

Comment:
We might not need the line `configLoader.getInitialConfig();`
We are not using the return value here.

Reply:
Address the same @VardhanThigle